docs/manual/common/guide/logging/code/log4j2-lagom-default.xml

```diff
@@ -12,7 +12,7 @@
 
     <Console name="STDOUT">
       <PatternLayout>
-        <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z'}{GMT+0} %highlight{%level} %logger{0} [%mdc] - %msg%n</Pattern>
+        <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z'}{GMT+0} %highlight{%level} %logger{10} [%mdc] - %msg%n</Pattern>
```

Comment:
Oh I see now... but this setting isn't about the lines of the stack trace, it's about the logger name.

The lines of the stack trace were previously configured with `%xException{10}`, which is still missing here.